src/utils.ts

```diff
@@ -452,13 +467,130 @@ export class Utils {
         }
         return;
     }
+
+    /**
+     * Generates GitHub workflow Summary
+     * This function runs as part of post-workflow cleanup function,
+     * collects existing section markdown files generated by the CLI,
+     * and constructs a single markdown file, to be displayed in the GitHub UI.
+     */
+    public static async generateWorkflowSummaryMarkdown() {
+        // This is the file path that the markdown should be copied to.
+        // In order for GitHub to display this as a job summary in the UI.
+        const githubStepSummaryFilePath: string | undefined = process.env.GITHUB_STEP_SUMMARY;
+        if (!githubStepSummaryFilePath) {
```

Comment:
Are you sure it can be empty?

Reply:
It shouldn't, but every env could potentially be empty.